aws/data_source_aws_kms_alias_test.go

```diff
@@ -23,8 +22,10 @@ func TestAccDataSourceAwsKmsAlias_AwsService(t *testing.T) {
 				Config: testAccDataSourceAwsKmsAlias_name(name),
 				Check: resource.ComposeTestCheckFunc(
 					testAccDataSourceAwsKmsAliasCheckExists(resourceName),
+					//lintignore:AWSAT001
 					resource.TestMatchResourceAttr(resourceName, "arn", regexp.MustCompile(fmt.Sprintf("^arn:[^:]+:kms:[^:]+:[^:]+:%s$", name))),
 					resource.TestCheckResourceAttr(resourceName, "name", name),
+					//lintignore:AWSAT001
 					resource.TestMatchResourceAttr(resourceName, "target_key_arn", regexp.MustCompile(fmt.Sprintf("^arn:[^:]+:kms:[^:]+:[^:]+:key/[a-z0-9]{8}-([a-z0-9]{4}-){3}[a-z0-9]{12}$"))),
```

Comment:
Nit: These can still be shortened so they don't require the ignores:

```suggestion
					testAccCheckResourceAttrRegionalARN(resourceName, "arn", "kms", name),
					resource.TestCheckResourceAttr(resourceName, "name", name),
					testAccMatchResourceAttrRegionalARN(resourceName, "target_key_arn", "kms", regexp.MustCompile(`key/[a-z0-9]{8}-([a-z0-9]{4}-){3}[a-z0-9]{12}`)),
```